wiki/Tournaments/Template/en.md

```diff
@@ -0,0 +1,144 @@
+---
+tags:
+- {ABBR}
+---
+
+<!--
+find-replace these curly braces then delete this comment
+{TOURNAMENT_NAME} = tournament name
+{MODE} = game mode
+{ABBR} = abbreviation of the title followed by its iteration/year occurence i.e. MWC 4K 2018
+{ORGANISER} = organiser of the tournament
+{NTH} = installment ordinal number
+{TOPIC_NUM} = forum thread number
+{STATS_LINK} = link to overall statistics (should be Google Drive link)
+You will still need to add other items such as tables, etc.
+Use .png as a default format for picture(s), since .jpg can't handle transparency
+-->
+
+# {TOURNAMENT_NAME}
+
+![{ABBR} Logo](img/logo.png)
+
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the {ORGANISER}. It is the {NTH} installment of the {TOURNAMENT_NAME}.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-01-01/2018-01-28 |
+| Live Drawings | 2018-01-29 12:00:00 UTC |
+| Group Stage |  |
+| Round of 16 |  |
+| Quarterfinals |  |
+| Semifinals |  |
+| Finals - Week 1 |  |
+| Finals - Week 2 |  |
+
+<!-- Use YYYY-MM-DD Format -->
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for {TOURNAMENT_NAME} winners. As these items may change with every installment of the tournament, it is only possible to give an estimate prize list for each {TOURNAMENT_NAME}.
```

Comment:
`As these items may change with every installment of the tournament, it is only possible to give an estimate prize list for each {TOURNAMENT_NAME}.` ->
`It is possible that these items change with every installment of the {TOURNAMENT_NAME}.`